.gitignore

```diff
@@ -80,6 +80,7 @@ $RECYCLE.BIN/
 .idea
 nbproject
 .vscode/
+.eslintcache
```

Comment:
This isn't needed anymore, but I'll leave it because it's nice to have in case someone runs eslint --cache manually